src/components/pages/places-list/places-list.js

```diff
@@ -55,10 +61,55 @@ export default class PlacesList extends Component {
     }
   }
 
+  async filters() {
+    try {
+      const places = await PlacesApi.getAllPlaces();
+
+      const filteredPlaces = places.filter(place => {
+        return (
+          this.state.regionDbId !== null
+            ? (place.isModerated === true && place.approved === true && place.regionId === this.state.regionDbId)
+            : (place.isModerated === true && place.approved === true)
```

Comment:
ADVICE:
If you write a condition which must be true not necessary to write in this way: place.isModerated === true ,
  place.isModerated will be enough